pkg/storage/utils/eosfs/eosfs.go

```diff
@@ -1450,40 +1270,23 @@ func (fs *eosfs) createNominalHome(ctx context.Context) error {
 		return errors.Wrap(err, "eosfs: error verifying if user home directory exists")
 	}
 
-	err = fs.createUserDir(ctx, u, home, false)
-	if err != nil {
-		err := errors.Wrap(err, "eosfs: error creating user dir")
-		return err
-	}
-
 	// set quota for user, depending on its type
 	quotaBytes := fs.conf.DefaultQuotaBytes
 	if u.Id.Type != userpb.UserType_USER_TYPE_PRIMARY {
 		quotaBytes = fs.conf.DefaultSecondaryQuotaBytes
 	}
-	quotaInfo := &eosclient.SetQuotaInfo{
-		Username:  u.Username,
-		UID:       auth.Role.UID,
-		GID:       auth.Role.GID,
-		MaxBytes:  quotaBytes,
-		MaxFiles:  fs.conf.DefaultQuotaFiles,
-		QuotaNode: fs.conf.QuotaNode,
-	}
-
-	err = fs.c.SetQuota(ctx, rootAuth, quotaInfo)
-	if err != nil {
-		err := errors.Wrap(err, "eosfs: error setting quota")
-		return err
-	}
 
-	if fs.conf.EnablePostCreateHomeHook {
-		if err := fs.runPostCreateHomeHook(ctx); err != nil {
+	if fs.conf.CreateHomeHook != "" {
```

Comment:
And the CreateHomeHook will be a local call to Manos' script then?

Reply:
Yes! Eventually to be replaced by a call to an HTTP endpoint.